lib/db/ReportDAO.js

```diff
@@ -12,9 +12,8 @@ const LIMIT = 100;
 
 class ReportDAO {
   static getReportsForStudy(study, reportFormat) {
-    const { countGroupId, type } = study;
-    const { id: categoryId, studyType } = type;
-    const id = `${categoryId}/${countGroupId}`;
+    const { countGroupId, studyType } = study;
+    const id = `${studyType.name}/${countGroupId}`;
```

Comment:
we should test this change with previously-submitted "Export Reports" jobs - it's possible that changing the report IDs here will break those.

Reply:
will test in dev; if there's work here, will add to #972